be/src/runtime/primitive_type.h

```diff
@@ -122,7 +122,6 @@ inline bool is_scalar_primitive_type(PrimitiveType ptype) {
     case TYPE_DECIMAL32:  /* 24 */
     case TYPE_DECIMAL64:  /* 25 */
     case TYPE_DECIMAL128: /* 26 */
-    case TYPE_JSON:       /* 27 */
```

Comment:
In FE, json is scalar_primitive_type. why BE not?

Reply:
fixed.